components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIImportUtil.java

```diff
@@ -932,4 +941,84 @@ private static void updateAPIWithCertificate(JsonElement certificate, APIProvide
             log.error(errorMessage, e);
         }
     }
+
+    /**
+     * This method adds API sequences to the imported API. If the sequence is a newly defined one, it is added.
+     *
+     * @param pathToArchive location of the extracted folder of the API
+     */
+    private static void addSOAPToREST(String pathToArchive, API importedApi, Registry registry)
+            throws APIImportExportException {
+
+        String inFlowFileLocation = pathToArchive + File.separator + "SoapToRest" + File.separator + "in";
```

Comment:
Shall we use constants for "in", "out", "SoapToRest" strings?

Reply:
resolved by #8332